src/widgets/splits/SplitHeader.cpp

```diff
@@ -537,51 +537,69 @@ std::unique_ptr<QMenu> SplitHeader::createMainMenu()
                 h->getDisplaySequence(HotkeyCategory::Split, "openViewerList"));
         }
 
-        moreMenu->addAction("Subscribe", this->split_, &Split::openSubPage);
+        moreMenu->addAction(
+            "Subscribe", this->split_, &Split::openSubPage,
+            h->getDisplaySequence(HotkeyCategory::Split, "subscribe"));
 
-        auto *action = new QAction(this);
-        action->setText("Notify when live");
-        action->setCheckable(true);
-
-        auto notifySeq = h->getDisplaySequence(
-            HotkeyCategory::Split, "setChannelNotification", {{"toggle"}});
-        if (notifySeq.isEmpty())
         {
```

Comment:
Please undo this. This makes the diff look more complicated.

Reply:
The code looked kinda confusing to me, as the blocks weren't doing logic isolation, had some extra code, and were repeating the condition. I'll put it in a separate cleanup PR, if you prefer that.